lib/biran/exceptions.rb

```diff
@@ -0,0 +1,16 @@
+module Biran
```

Comment:
Not 100% sure this file is correct or done the best way, but seems to be working.